Jenkinsfile

```diff
@@ -37,18 +37,19 @@ node('vetsgov-general-purpose') {
           if (!shouldBuild) { return }
 
           try {
+            // try to build using fresh drupal content
             commonStages.build(ref, dockerContainer, assetSource, envName, false, contentOnlyBuild)
             envUsedCache[envName] = false
           } catch (error) {
-            if (!contentOnlyBuild) {
+            try {
+              // try to build again using the cached drupal content
               dockerContainer.inside(DOCKER_ARGS) {
                 sh "cd /application && node script/drupal-aws-cache.js --fetch --buildtype=${envName}"
               }
               commonStages.build(ref, dockerContainer, assetSource, envName, true, contentOnlyBuild)
               envUsedCache[envName] = true
-            } else {
```

Comment:
This `else` block should be unreachable because we shouldn't be building for content-only builds. 

```
shouldBuild = !contentOnlyBuild || envName == params.cmsEnvBuildOverride
if (!shouldBuild) { return }

if (!contentOnlyBuild) {
...
} else {
// this should be unreachable because of the early return when `shouldBuild` is falsy
}
```

Reply:
I don't think this is unreachable. This else block would apply when it is a content build and the `envName` matches the override. The early return is when this doesn't match the override.

Reply:
> I don't think this is unreachable. This else block would apply when it is a content build and the `envName` matches the override. The `shouldBuild` condition says we should build when it's `not a content build` or `a content build for this environment`. The early return occurs for content-only builds when `envName` doesn't match that override.

Good call. I'll update my changes.

Reply:
I reverted my `Jenkinsfile` changes in 92347145c57335ed23db5ed55a6efd9f0dcafef2, and started over from scratch.